### PR TITLE
Milestone 8: Capybara Integration Tests and N+1 Issues

### PR DESCRIPTION
**Pull request by SaidRasinlic**
###  ♻️ REQUIRED CHANGES (Fix spec/features tests with Capybara): 

**I've successfully fixed spec/features test errors, please make sure to git pull changes and bundle install newly added gem for Database cleaning, and then you can run the command "rspec spec/features"**
<hr/>

### 🚀 In this milestone I've completed the following activities:

- Wrote integration tests for each view.
- Fixed N+1 issues in controllers when fetching all posts and their comments for a user.

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -53,6 +53,31 @@ a {
   align-self: center;
 }
 
+.log-out {
```
Comment (GzimAsani) [inline app/assets/stylesheets/application.css:56]: - [x] I am getting an error when trying to run your project coming from this path Blog-App-II/app/assets/config/manifest.js
image

Comment (GzimAsani) [reply app/assets/stylesheets/application.css:56]: - [x] Your tests are failing, kindly update them
image
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @SaidRasinlic         :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your tests, they covered everything:+1:  
 
 ## Required Changes 

- [ ] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (SaidRasinlic) [reply app/assets/stylesheets/application.css:56]: #### All issues are fixed now! Thank you 😃!

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Said,  

Your project is complete, great job🥇! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- N +1 issue properly handled
- Good coverage of views integration specs


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
- Kindly add instructions in your ReadMe on how to run your app, as well as run spec test cases of your project under the _Getting Started_ section.
- Kindly fix broken screenshots link in your ReadMe
- I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor. 
Implementing the above change(s) will make your ReadMe appear more professional
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 
<br><br>
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**